ReactWindows/ReactNative/Views/View/ReactViewManager.cs

```diff
@@ -0,0 +1,270 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.Storage;
+using Windows.Storage.AccessCache;
+using Windows.ApplicationModel.DataTransfer;
+using ReactNative.UIManager.Annotations;
+using ReactNative.UIManager;
+using Newtonsoft.Json.Linq;
+using ReactNative.UIManager.Events;
+
+namespace ReactNative.Views.View
+{
+    /// <summary>
+    /// The UWP-specific RCTView implementation.
+    /// </summary>
+    public class ReactViewManager: BaseViewManager
+    {
+        private class DragDropEvent : Event
+        {
+            private readonly string _name;
+            private readonly JObject _data;
+
+            public override string EventName => _name;
+            public override bool CanCoalesce => false;
+
+            public DragDropEvent(int viewTag, string name, JObject data)
+                : base(viewTag)
+            {
+                _name = name;
+                _data = data;
+            }
+
+            public override void Dispatch(RCTEventEmitter eventEmitter)
+            {
+                eventEmitter.receiveEvent(ViewTag, EventName, new JObject
+                {
+                    { "target", ViewTag },
+                    { "dataTransfer", _data }
+                });
+            }
+        }
+
+        /// <summary>
+        /// The exported custom bubbling event types.
+        /// </summary>
+        public override IReadOnlyDictionary<string, object> ExportedCustomBubblingEventTypeConstants
```

Comment:
>ExportedCustomBubblingEventTypeConstants [](start = 60, length = 40)

If you move the AllowDrop behavior to BaseViewManager, we can move this event metadata to UIManagerModule.Constants.cs.

Reply:
Indeed. Fixed.